lib/rest_client.js

```diff
@@ -126,10 +126,11 @@ module.exports.RestClient = function (options) {
     return apiCall(request_data, request_token);
   }
 
-  instance.delete = function (resourceUrl, request_token = '') {
+  instance.delete = function (resourceUrl, data, request_token = '') {
     const request_data = {
       url: createUrl(resourceUrl),
-      method: 'DELETE'
+      method: 'DELETE',
+      body: data
```

Comment:
It's not recommended to add payload to DELETE so lets make it optional and move decision to devs
```
const request_data = {
    url: createUrl(resourceUrl),
    method: 'DELETE'
}
if (data) {
    request_data.body = data
}
```